### ISSUE TITLE
visualizer with different tasks

### ISSUE THREAD

**Issue by xianglinyang** [xianglinyang/DeepDebugger#9]
multiclass classification
[logit+sigmoid]*m

single class classification: 
1. logits+softmax
2. logit+sigmoid
